src/components/Wallet/index.jsx

```diff
@@ -222,6 +238,12 @@ class CreateWallet extends React.Component {
     this.setState({ refreshing: false });
   };
 
+  resetErrorAndClearTargetValue = (event) => {
+    // eslint-disable-next-line no-param-reassign
+    event.target.value = "";
```

Comment:
This feels a little weird to me and wondering if it could actually cause a problem. Wouldn't the value be getting determined by something in the state already? The right way to update input/form values is through the state of the component rather reassigning it directly.

Reply:
this was a bugfix, maybe a better way to do it. bc we're using `onChange` -- if you select a file and it has an error inside it ... then you go fix the error in the file and re-select the same file, the event will not fire again. I believe this is because the target.value is the same filename.